src/Select/Select.js

```diff
@@ -0,0 +1,207 @@
+// @flow
+import * as React from "react";
+import defaultTokens from "@kiwicom/orbit-design-tokens";
+import styled from "styled-components";
+
+import FormLabel from "../FormLabel/FormLabel";
+import ChevronDown from "../icons/ChevronDown";
+import FormFeedback, { TYPE_OPTIONS } from "../FormFeedback/FormFeedback";
+
+export const SIZE_OPTIONS = {
+  SMALL: "small",
+  NORMAL: "normal",
+};
+
+type Option = {
+  value: string | number,
+  label?: string,
+  disabled?: boolean,
+};
+
+type Props = {
+  size: $Values<typeof SIZE_OPTIONS>,
+  label?: string,
+  placeholder?: string,
+  value: string,
+  disabled?: boolean,
+  error?: React.Node,
+  help?: React.Node,
+  onChange?: (SyntheticInputEvent<HTMLSelectElement>) => void,
+  onFocus?: (SyntheticInputEvent<HTMLSelectElement>) => void,
+  onBlur?: (SyntheticInputEvent<HTMLSelectElement>) => void,
+  options: Option[],
+  prefix?: React.Node,
+  theme: typeof defaultTokens,
+};
+
+const StyledSelect = styled(({ className, children, value, onChange, onFocus, onBlur }) => (
+  <select value={value} className={className} onChange={onChange} onFocus={onFocus} onBlur={onBlur}>
+    {children}
+  </select>
+))`
+  appearance: none;
+  background: ${({ theme }) => theme.backgroundInput};
+  border: 1px solid;
+  border-radius: ${({ theme }) => theme.borderRadiusNormal};
+  cursor: pointer;
+  color: ${({ theme, filled }) => (filled ? theme.paletteInkLight : theme.colorTextInput)};
+  font-family: ${({ theme }) => theme.fontFamily};
+  font-size: ${({ theme, size }) =>
+    size === SIZE_OPTIONS.SMALL ? theme.fontSizeInputSmall : theme.fontSizeInputNormal};
+  height: ${({ theme, size }) =>
+    size === SIZE_OPTIONS.SMALL ? theme.heightInputSmall : theme.heightInputNormal};
+  padding: ${({ theme, size }) =>
+    size === SIZE_OPTIONS.SMALL
+      ? `0 ${theme.spaceXLarge} 0 ${theme.spaceSmall}`
+      : `0 ${theme.spaceXXLarge} 0 ${theme.spaceSmall}`};
+  outline: none;
+  width: 100%;
+
+  &::-ms-expand {
+    display: none;
+  }
+
+  /* With prefix */
+  padding-left: ${({ prefix }) => prefix && "48px"}; // TODO: Create token
+
+  /* Based on state of select */
+  border-color: ${({ theme, error }) =>
```

Comment:
```css
border: 1px solid
```
and delete line 44

Reply:
It defines basic style of border, so I want to let it

Reply:
Ok :)